### PR TITLE
Pre split no 6 no knn

### PR DESCRIPTION
**Pull request by CharlesGaydon**

### CODE REVIEW

**Comment by CharlesGaydon** [comment]
Many changes here : 
- We use pre-splitted LAS instead of full LAS to accelerate training. This happens in `bash/data_preparation/split_clouds.sh`
- The best model configuration was reproduced in this setting, with equivalent results.
- We remove KNN interpolation before calculation of losses / metrics. This speeds up training. This has a small impact on measure of performances since we subsample with a high fidelity (12500pts/50m*50m).
- The use of GPU in RandLa-Net is corrected: there were many unnecessary migrations to CPU at random places, and lack of migration to GPU at other places. MLoving the model to GPU is now fully handled by Pytorch Lightning OR done manually in `predict.py`
- Use of several test/val dataloaders was finally removed since Lightning does not support reduction of loss across dataloaders.
